Makefile

```diff
@@ -180,6 +180,7 @@ killall:
 
 clean:
 	@./rebar3 clean
+	@rm -rf _build/
```

Comment:
`( cd $(HTTP_APP) && $(MAKE) clean; )` ?

Reply:
and that for all directories in which we have a make file, I guess...